lib/log_displayer.js

```diff
@@ -41,6 +42,27 @@ function colorize (line) {
 }
 
 function readLogs (logplexURL) {
+  let u = url.parse(logplexURL)
+  if (u.query && u.query.indexOf('srv') !== -1) {
```

Comment:
The query is expected to be empty for the new URLs, right? This seems reasonable.

Reply:
Yep! I think it's simpler, without having to hit a remote resource, and incur additional overhead.

Reply:
minor nit: you can use `u.query.includes('src')` in the node version we support (node 9)

Reply:
Nice TIL. Updated